setupMYM.m

```diff
@@ -0,0 +1,67 @@
+function setupMYM(version, force)
+
+    if nargin < 1
+        version = 'master';
+    elseif nargin < 2
+        force = false;
+    end
+    
+    persistent INVOKED;
+    
+    if ~isempty(INVOKED) && ~force
+        return
+    end
+
+    base = fileparts(mfilename('fullpath'));
+
+    mymdir = fullfile(base, 'mym');
+    
+    if isdir(mymdir)
+        if force
+            fprintf('force install.. removing %s\n', mymdir);
+            rmdir(mymdir, 's');
+        else
+            fprintf('Warning: mym directory exists. not re-installing.\n');
+            fprintf('  to override, pass force=true\n');
+        end
+    end
+
+    if ~isdir(mymdir) %% mym directory missing, download and install
```

Comment:
messages adjusted; handling of the existing directory / force / etc logic pushed up to higher block. 
marking this as resolved but noting since it was commented on an affected line which is now different.